articles/aks/coredns-custom.md

```diff
@@ -162,6 +181,13 @@ kubectl apply -f corednsms.yaml
 kubectl delete pod --namespace kube-system --selector k8s-app=kube-dns
 ```
 
+As in the previous examples, create the ConfigMap using the [kubectl apply configmap][kubectl-apply] command and specify the name of your YAML manifest. Then, force CoreDNS to reload the ConfigMap using the [kubectl delete pod][kubectl delete] for the Kubernetes Scheduler to recreate them:
```

Comment:
dupe? looks identical to the previous section.

BTW, delete coredns pod is not necessary. coredns will auto reload the config within 1 minute.
Even if you want to force delete, please recommend to use kubectl rollout restart command

```
kubectl rollout restart --namespace kube-system deployment/coredns
```